src/client/containers/Home/Home.js

```diff
@@ -1,3 +1,49 @@
 import React from 'react';
+import './Home.styles.css';
+import '../../index.css';
+import { Link } from 'react-router-dom';
+// eslint-disable-next-line import/no-useless-path-segments
+import logoAnna from '../../../client/assets/images/logoAnna.png';
```

Comment:
```suggestion
import logoAnna from './../../../client/assets/images/logoAnna.png';
```

This does not work on the remote, try what i suggested above. If that does not work try this:

`import logoAnna from './../../client/assets/images/logoAnna.png';`